Parking/accessible_parking.csv

```diff
@@ -0,0 +1,31 @@
+name,description,note,latitude,longitude
+O,Optometry Building (OPT),Permit,43.47582060865965,-80.54503210633384
+,Student Village 1 (V1); William Lyon Mackenzie King Village (MKV),,43.47111968920361,-80.55104258620942
+,Student Village 1 (V1); University Club (UC),,43.47211149208808,-80.54873834383558

```

Comment:
I think it would be a good idea if all parking files have a `type` column that contains the service info (metered, accessible, visitor, etc).